testdata/ostrusted-inferrable-test/WildcardsProblem.java

```diff
@@ -1,7 +1,6 @@
-//@skip-test
 class WildcardsProblem<A>{
     A field;
-    public void method(WildcardsProblem t){
+    public void method(WildcardsProblem<?> t){
         Object t_field = t.field;
     }
 }
```

Comment:
DebugSolver's output for this test case:
```
Created ConstantSlot
  1 @OsTrusted

  2 @OsUntrusted

  3 @PolyOsTrusted

Created SourceVariableSlot
  4
  AstPath: MissingLocation

  5: merged to -> [LubVariableSlot(17)]
  AstPath: AstPathLocation( WildcardsProblem.null.null:WildcardsProblem:::Class.typeParameter 0 )

  6: merged to -> [LubVariableSlot(16)]
  AstPath: AstPathLocation( WildcardsProblem.null.null:WildcardsProblem:::Class.typeParameter 0, TypeParameter.bound 0 )

  7
  AstPath: ClassDeclLocation( WildcardsProblem )

  8
  AstPath: AstPathLocation( WildcardsProblem.null.field:WildcardsProblem::field:Variable.type )

  11
  AstPath: AstPathLocation( WildcardsProblem.<init>()V.null:WildcardsProblem:<init>()V:: )

  12
  AstPath: AstPathLocation( WildcardsProblem.method(LWildcardsProblem;)V.null:WildcardsProblem:method(LWildcardsProblem;)V::Method.parameter 0, Variable.type, ParameterizedType.type )

  13: merged to -> [LubVariableSlot(17)]
  AstPath: AstPathLocation( WildcardsProblem.method(LWildcardsProblem;)V.null:WildcardsProblem:method(LWildcardsProblem;)V::Method.parameter 0, Variable.type, ParameterizedType.typeArg
ument 0 )

  14: merged to -> [LubVariableSlot(16)]
  AstPath: MissingLocation

  15
  AstPath: AstPathLocation( WildcardsProblem.method(LWildcardsProblem;)V.null:WildcardsProblem:method(LWildcardsProblem;)V::Method.body, Block.statement 0, Variable.type )

  21
  AstPath: AstPathLocation( WildcardsProblem.method(LWildcardsProblem;)V.null:WildcardsProblem:method(LWildcardsProblem;)V::Method.parameter -1, ParameterizedType.typeArgument 0 )

  22
  AstPath: AstPathLocation( WildcardsProblem.method(LWildcardsProblem;)V.null:WildcardsProblem:method(LWildcardsProblem;)V::Method.parameter -1 )

Created ExistentialVariableSlot
  9: ( 8 | 6 )
  AstPath: AstPathLocation( WildcardsProblem.null.field:WildcardsProblem::field:Variable.type )

  10: ( 8 | 5 )
  AstPath: AstPathLocation( WildcardsProblem.null.field:WildcardsProblem::field:Variable.type )

  18: ( 8 | 16 )
  AstPath: AstPathLocation( WildcardsProblem.null.field:WildcardsProblem::field:Variable.type )

  19: ( 8 | 17 )
  AstPath: AstPathLocation( WildcardsProblem.null.field:WildcardsProblem::field:Variable.type )

Created LubVariableSlot
  16: lub-of [SourceVariableSlot(6), SourceVariableSlot(14)]

  17: lub-of [SourceVariableSlot(5), SourceVariableSlot(13)]

Created RefinementVariableSlot
  20[ ↧ 15
  AstPath: AstPathLocation( WildcardsProblem.method(LWildcardsProblem;)V.null:WildcardsProblem:method(LWildcardsProblem;)V::Method.body, Block.statement 0, Variable.type ) ]
  AstPath: AstPathLocation( WildcardsProblem.method(LWildcardsProblem;)V.null:WildcardsProblem:method(LWildcardsProblem;)V::Method.body, Block.statement 0 )

Created these Constraints:
  11 <: 7

  12 <: 7

  13 <: 14

  13 <: 17

  16 <: 14

  16 <: 6

  5 <: 17

  5 <: 6

  if ( 8 exists ) {
    20[ ↧ 15 ] == 8
  } else {
    20[ ↧ 15 ] == 6
  }
```

and we get this annotated source:
```
import checkers.inference.qual.VarAnnot;
@VarAnnot(7)
class WildcardsProblem<@VarAnnot(5) A extends @VarAnnot(6) Object>{
    @VarAnnot(8)
    A field;
    public void method(@VarAnnot(22) WildcardsProblem<@VarAnnot(21) A> this, @VarAnnot(12) WildcardsProblem<@VarAnnot(13) ?> t){
        Object t_field = t.field;
    }
}
```

It seems 16 is the GLB of 14 and 6, but the invocation of `greatestLowerBound` is from some internal logics in the Checker Framework. We may want to verify if this behavior make sense?